js/script.js

```diff
@@ -0,0 +1,6 @@
+let add = (function() {
+  let total = 0;
+  return function(x) {
+    return total += Number(x);
+  }
```

Comment:
I'd rather all of the logic for this comes from a function you call in the HTML and then the function handles the alert. This code is pretty complex, and can be simplified a lot.  Plus the syntax looks like an IIFE, not sure if that's the best way to handle this.

We're not launching rockets, simplicity is preferred.